client/src/pages/ModalBookingScreen.js

```diff
@@ -141,22 +149,27 @@ function ModalBookingScreen({ bookingDate, refreshBooking, desks }) {
 									checked={dontSelectDesk}
 								/>
 							</Form.Field>
-							{deskErrorMessage && (
-								<p style={{ color: "maroon" }}>{deskErrorMessage}</p>
-							)}
 							{!dontSelectDesk && (
-								<div>
+								<Form.Field>
+									<Message
+										error
+										content={deskErrorMessage}
+										style={{ width: "200px" }}
+									/>
 									<Dropdown
 										error={!!deskErrorMessage}
 										placeholder="Desk Selection"
 										options={deskSelection}
 										selection
 										value={deskId}
-										onChange={(e, data) => setDeskId(data.value)}
+										onChange={(e, data) => {
+											setDeskErrorMessage(null);
```

Comment:
I liked that you implemented Greg's observation. By the way, I'm learning a lot reviewing your code